.ci/containers/gcb-terraform-vcr-tester/Dockerfile

```diff
@@ -12,8 +12,8 @@ ENV PATH $PATH:/root/google-cloud-sdk/bin
 
 ADD test_terraform_vcr.sh /test_terraform_vcr.sh
 
-RUN wget https://releases.hashicorp.com/terraform/1.1.8/terraform_1.1.8_linux_amd64.zip
-RUN unzip terraform_1.1.8_linux_amd64.zip && rm terraform_1.1.8_linux_amd64.zip
+RUN wget https://releases.hashicorp.com/terraform/1.2.5/terraform_1.2.5_linux_amd64.zip
```

Comment:
We could consider using a multi-stage build rather than https://github.com/GoogleCloudPlatform/magic-modules/blob/e90865120d260db594bb3873fd032e4311437ecc/.ci/containers/downstream-builder/Dockerfile#L12, that gets us out of some of the low-level file management here.

Reply:
hmm I'm not a super big fan of that approach. This is pretty lightweight. No reason to pull a container image only for a one-liner.